src/jobs/adapters/job.adapter.ts

```
@@ -0,0 +1,31 @@
+      updatedAt: job.updatedAt,
+      // Add any other fields you need to transform or include
+    };
+  }
+
+  /**
+   * Convert an array of Job entities to the format expected by PaginatedJobResponseDto
+   */
+  static toJobPostingEntities(jobs: Job[]): any[] {
+    return jobs.map((job) => this.toJobPostingEntity(job));
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)